frontend/src/app/dashboard/components/Playground/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './Playground'
```

Comment:
Let's keep the standard of the code base, where we have the components on the index.tsx files